modules/alloy/src/demo/ts/ephox/alloy/demo/SliderDemo.ts

```diff
@@ -171,13 +171,13 @@ export default (): void => {
 
       onChange: (_slider, thumb, value: SliderValue) => {
         if (isValueX(value)) {
-          setColor(thumb, value.x);
+          setColor(thumb, (value as SliderValueXY).x);
         }
       },
 
       onInit: (_slider, thumb, _spectrum, value: SliderValue) => {
         if (isValueX(value)) {
```

Comment:
Likewise here, as these were only needed as a type guard to make the types play nice and aren't really useful if we're just casting.